classes/class-wc-connect-nux.php

```diff
@@ -149,9 +166,7 @@ public function register_order_page_labels_pointer( $pointers ) {
 
 			// If the user is not new to labels, we should just dismiss this pointer
 			if ( ! $this->is_new_labels_user() ) {
-				$dismissed_pointers[] = 'wc_services_labels_metabox';
```

Comment:
Do we still need `$dismissed_pointers` declared above, considering that the pointers are checked for having been dismissed at [a later point](https://github.com/Automattic/woocommerce-services/blob/02cad08415eca8b1321dff4f037e52cda38917ee/classes/class-wc-connect-nux.php#L94-L96) anyway?

Reply:
Nice catch - I'll clean it up.

Reply:
Fixed in bdbdd2fc.